examples/vue-example-app/src/sdk/index.ts

```diff
@@ -1,26 +1,24 @@
-import fetch from 'node-fetch'
-import { ClientBuilder } from '@commercetools/sdk-client-v2'
+import { ClientBuilder } from '@commercetools/ts-client'
 import { createApiBuilderFromCtpClient } from '@commercetools/platform-sdk'
 
-export const projectKey = process.env.VUE_APP_CTP_PROJECT_KEY
+export const projectKey = import.meta.env.VITE_CTP_PROJECT_KEY
 const authMiddlewareOptions = {
   host: 'https://auth.europe-west1.gcp.commercetools.com',
```

Comment:
in baase of task related to the regions this can be passed through the env variables or we can do it in the future. Plese check datadog or newrelic demo apps